packages/swap/src/event-handlers/index.ts

```diff
@@ -112,6 +116,10 @@ const handlers = [
           name: events[`${chain}IngressEgress`].CcmBroadcastRequested,
           handler: networkCcmBroadcastRequested,
         },
+        {
+          name: events[`${chain}IngressEgress`].DepositReceived,
+          handler: depositReceived(chain),
```

Comment:
we prefixed the other ingress-egress events with `network` so this would be `networkDepositReceived`